robotmpcs/planner/mpcPlanner.py

```diff
@@ -2,13 +2,22 @@
 import yaml
 import os
 import sys
+sys.path.append("../")
+sys.path.append("")
```

Comment:
This arbitrary adding of paths is bad practice. the function`load_forces_path` can simply be inside robotmpcs.

Reply:
removed